tools/AutoTuner/README.md

```diff
@@ -0,0 +1,205 @@
+# AutoTuner with Ray cluster at GCP
+
+## Enable GKE
+
+Follow the Google quickstart guide up to the section "Create a GKE cluster"
+[here](https://cloud.google.com/kubernetes-engine/docs/quickstart). The
+quickstart guide will show how to enable GKE (Google's Kubernetes Engine)
+start a CLI interface and get the settings for your project.
+
+## Create a Kubernetes cluster
+
+Create GKE cluster using the following `gcloud` command as a guide.
+Note that each argument defines a characteristic of the cluster, further
```

Comment:
Change to: Note that each argument defines a characteristic of the cluster. Furthermore,
the cluster and node pool allocate resources that cost money. Hence, so be
mindful when choosing the configuration. For additional information see:

Reply:
Fixed